core-service/pom.xml

```diff
@@ -45,31 +48,32 @@
             <artifactId>postgresql</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
```

Comment:
https://springframework.guru/using-mapstruct-with-project-lombok/

Reply:
Thank you, read this article, useful material